go.mod

```diff
@@ -4,7 +4,8 @@ go 1.16
 
 require (
 	github.com/gorilla/mux v1.8.0
+	github.com/grafana/loki v1.6.1
```

Comment:
More a question here, what makes these requests different that we now need loki dependency while we did not need before?

Reply:
Well that's not mandatory but importing `loki` allow you to reflect all types from [loghttp](https://github.com/jpinsonneau/network-observability-console-plugin/blob/e15f457bde68e2af782edf9afb1836d4b20be482/vendor/github.com/grafana/loki/pkg/loghttp/query.go#L35) that contains `QueryResponse` `ResultValue` `Streams` `LabelSet` `Entry`

There is also some interesting functions to [Unmarshal](https://github.com/jpinsonneau/network-observability-console-plugin/blob/e15f457bde68e2af782edf9afb1836d4b20be482/vendor/github.com/grafana/loki/pkg/loghttp/query.go#L108) managing all types available in loki

Reply:
BTW it was not needed before because we simply forward bytes from `loki` response to `front end` without reading it except if there is an error code.

Reply:
Ok, thanks, for the explanations!